test/e2e/suite_test.go

```diff
@@ -92,12 +94,15 @@ var _ = SynchronizedBeforeSuite(func(ctx context.Context) []byte {
 	Expect(err).NotTo(HaveOccurred())
 	Expect(cfg.Host).To(Equal(string(host)))
 
-	err = ytv1.AddToScheme(scheme.Scheme)
+	scheme := runtime.NewScheme()
```

Comment:
Could you please explain how this scheme is different from one registered in scheme package?

Reply:
Do you mean default global scheme?
I'm not sure that it is thread safe.
It's cleaner to build fresh scheme here.